src/components/IconButton/IconButton.tsx

```diff
@@ -42,13 +43,16 @@ export interface IconButtonProps extends VibeComponentProps {
    * Size of the icon
    */
   size?: Size;
-
   /**
    * Whether the tooltip should be displayed or not
    */
   hideTooltip?: boolean;
   /**
-   * Tooltip wraps the button icon, it will display in the tooltip, if not present the aria label will be shown
+   * Props for Tooltip wrapper component
+   */
+  tooltipProps?: Partial<TooltipProps>;
+  /**
+   * Backward compatibility for props naming
```

Comment:
why backward compatibility? in 99% of the cases developers will prefer to use this and not tooltip props. let's keep it and return the old comment.

Reply:
why this is still Backward compatibility for props naming ?  let's switch to old comment